var/spack/repos/builtin/packages/sensei/package.py

```diff
@@ -69,39 +69,38 @@ def cmake_args(self):
             '-DCMAKE_C_FLAGS=-fPIC -Wall -Wextra -O3 -mtune=generic'
         ]
 
-        if (spec.variants['catalyst'].value):
+        if 'catalyst' in spec:
             args.append('-DENABLE_CATALYST:BOOL=ON')
             args.append('-DENABLE_CATALYST_PYTHON:BOOL=ON')
-            args.append('-DParaView_DIR:PATH=={0}'.format(spec['paraview'].prefix))
+            args.append('-DParaView_DIR:PATH={0}' % spec['paraview'].prefix)
+        else:
+            args.append('-DENABLE_CATALYST:BOOL=OFF')
```

Comment:
It's generaly a good idea to use explicit on/off options rather than explicit on / implicit off.  It prevents options from accidentally getting enabled by detecting dependencies outside of spack by accident.  So instead of
```python
if 'foo' in spec:
  args.append('-Denable_bar=ON')
```
use
```python
if 'foo' in spec:
  args.append('-Denable_bar=ON')
else:
  args.append('-Denable_bar=OFF')
```

If it's just a single boolean value then you can simplify it even further with
```python
  args.append('-Denable_bar={0}' % str('foo' in spec))
```
since `str('foo' in spec)` evaluates to the strings `'True'` or `'False'` which are valid values to pass to cmake options.